### PR TITLE
Update task

### PR DESCRIPTION
**Pull request by 2000-27**

### CODE REVIEW

File: app/api/manager.py [hunk 1]
```diff
@@ -42,5 +42,5 @@ def update_task():
     msg = ""
     info_schema = InfoSchema()
     task = data_now_json_str(info_schema)
-    msg = update(task)
+   # msg = update(task)
```
Comment (pythexcel) [inline app/api/manager.py:45]: why this is commented?

File: app/dob.py [hunk 2]
```diff
@@ -53,25 +55,21 @@ def task_delete(task):
...
-def update_status(task):
-    filter = {'_id': ObjectId(task['task_id'])}
-    newvstatus = {"$set": {'status': task['status']}}  
-    mongo.db.tasks.update_one(filter, newvstatus)
-    msg = "task is update"
-    return msg    
+    keysList = list(task.keys())
+    print("mykeylist is ", keysList)
 
+    if 'new_task' in keysList:
```
Comment (pythexcel) [inline app/dob.py:63]: why do we need to write the same code multiple times?

File: app/api/employee.py [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
 
 
 @employee_bp.route('/view-task', endpoint='view_task',  methods=['GET'])
 @employee_required
 def view_task():
     message = ""
-    view_schema = ViewSchema()
     try:
-        employee_detail = data_now_json_str(view_schema)     
+        task_id = request.args.get('task_id',default="N/A")
```
Comment (Mohit-00-11) [inline app/api/employee.py:17]: Use None instead of "N/A"

File: app/api/employee.py [hunk 4]
```diff
@@ -1,38 +1,38 @@
...
+            message = "Invalid object id "
+            return jsonify({"message": message})
 
-    if task is None:
-        message = "no task is assign to him"
-        print("hello")
-        return jsonify({"message": message})
-    return jsonify({"message": str(task)})
+        if task is None:
+            message = "no task is assign to him"
```
Comment (Mohit-00-11) [inline app/api/employee.py:31]: The error message is not generic, What is employee is "Female" not "Male"

File: app/api/auth.py [hunk 5]
```diff
@@ -1,49 +1,58 @@
...
+    request_data = request.get_json()
+    login_schema = LoginSchema()
+    try:
+        result = login_schema.load(request_data)
+    except ValidationError as err:
+        return jsonify(err.messages), 400
+    data_now_json_str = dumps(result)
+    data = loads(data_now_json_str)
+    user_email = data['email']
+    if mongo.db.users.find_one({"email": user_email}) is None:
```
Comment (pythexcel) [inline app/api/auth.py:43]: Unwanted extra query due to an invalid condition

File: app/api/employee.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                   methods=['PATCH'])
+@employee_required
+def change_status():
+    message = ""
+    status_schema = StatusSchema()
+    try:
+        task = data_now_json_str(status_schema)
+        message = update(task)
+    except Exception as err:
+        return jsonify({"err": str(err)})
```
Comment (pythexcel) [inline app/api/employee.py:48]: make it common, The error response is different as of now.

**Pull request merged**